### PR TITLE
add function to combine accumulator update

### PR DESCRIPTION
**Pull request by cctdaniel**
add function to combine accumulator updates by:
1. iterate through the list of `update_data` and check their `vaa`
2. for those with the same `vaa`, combine them into one accumulator update by incrementing the number of updates in the first `update_data` and appending the update to `updates`
3. for those with different `vaa`, keep them
4. serialize them back to the accumulator `update_data` format

### CODE REVIEW

File: pythclient/price_feeds.py [hunk 1]
```diff
@@ -112,6 +115,54 @@ def to_dict(self, verbose=False, vaa_format=DEFAULT_VAA_ENCODING):
         return result
 
 
+class MerkleUpdate:
```
Comment (cctdaniel) [inline pythclient/price_feeds.py:118]: these names are based on the [Accumulator Wire Format document on Notion](https://www.notion.so/pyth-network/Accumulator-wire-format-3478fcc0eca94e768652d4403f36bc70?pvs=4)

File: pythclient/price_feeds.py [hunk 2]
```diff
@@ -502,28 +581,242 @@ def extract_price_info_from_accumulator_update(
...
+        serialized_data.append(update.proof_size)
+        for proof in update.proof:
+            serialized_data.extend(proof)
+    if encoding == "hex":
+        return serialized_data.hex()
+    else:
+        return base64.b64encode(serialized_data).decode("ascii")
+
+
+def parse_accumulator_update(update_data, encoding):
```
Comment (ali-bahjati) [inline pythclient/price_feeds.py:729]: this is quite similar to extract_price_infos_from_accumulator... Can't we remove the duplicate code? the former method can use the data on this method to get the price infos

**Review by ali-bahjati — COMMENTED** [review]
It is very nice! i think there are some duplicate codes that we can remove. I like to review again after you removed them.

File: pythclient/price_feeds.py (hunk 2 above)
Comment (cctdaniel) [reply pythclient/price_feeds.py:685]: you're right, updated ser

**Review by ali-bahjati — APPROVED** [review]
Very nice!

**Pull request merged**